### ISSUE TITLE
Minor issues

### ISSUE THREAD

**Issue by auxh** [finndev/PokeBuddy#137]
Okay, so Pokebuddy seems to spin the same pokestops and receive no xp for them. This will repeat as it doesn't go to other stops further away. What I see is that the red stops the are marked as looted become blue again for some odd reason and it returns to that marker. Also been hitting a lot of encounterfails and Pokemon escape? I would appreciate feedback thanks!

**Comment by IamDeric** [comment]
i think you got softban

**Comment by auxh** [comment]
No, the thing is .. I'll earn the experience for these four stops right? Which turn red to indicate that they have been looted. Then the map glitches and they revert back to blue which makes buddy run back them to see that they are on cool down. This has just started happening me with the new update. Also the encounterfailed was already addressed in another post and can't tell if it's being near a coast is what bugs it out. :/.

**Comment by IamDeric** [comment]
can you try to redownload the pokebuddy bot. and try it again, i mean delete your copy now and download a new one.

**Comment by auxh** [comment]
I've done so many times now but will give it another shot

**Comment by auxh** [comment]
To no avail sadly.

**Comment by auxh** [comment]
I've never seen this token expiration thing that requires reauthentication either. This is quite the conundrum. The stops will revert to blue after about 4 stops just to add on.

**Comment by whoppster** [comment]
I've seen this when moving the coordinates around too much in short time. Restarting and waiting ~5 minutes solves this for me every time so far.

**Comment by whoppster** [comment]
auxh, token expiration seems to happen if you have the game on our cellphone at the same time as running pokebuddy, had that issue twice before I realized that I had to logout from the cell.

**Comment by auxh** [comment]
Will give this a try. Let you guys know if it works!

**Comment by auxh** [comment]
The issue still persists sadly. I created a whole new google account to test and made sure I was logged out

**Comment by auxh** [comment]
Maybe it's just me that it doesn't accurately track which pokestops it has already hit. It's so bizarre.. they are red and next second they are blue when I know they are cooldown

**Comment by whoppster** [comment]
Hmm. I created two new PTCs yesterday and left them running, both are working fine. I placed them on different continents and chose country accordingly (Sweden and usa). I then accidentally logged in on my phone using the one that was set to USA and when I, approx 15 minutes later, started pokebuddy with the USA account again I didn't get any xp from stops and all Pokémon fled. After leaving it running for like 8 minutes, everything was back to normal again.

**Comment by whoppster** [comment]
When I logged in on my phone using the USA account, I was in Sweden. I could catch Pokémons on my phone and spin the stops but the problems began when I started pokebuddy after that. So this account was in new York, jumped to Sweden and the back to New York in a matter of minutes. I presume that Niantic has controls in place for these kind of things and that's what's causing this issue. But of course I can be wrong...

**Comment by auxh** [comment]
Thanks for the feedback. It is a possibility that Niantic would hit hotspots.
